src/Middleware/RecordTransaction.php

```diff
@@ -126,7 +126,7 @@ public function terminate($request, $response)
             $this->agent->send();
         }
         catch(\Throwable $t) {
-            Log::error(__METHOD__ . ' - ' . get_class($t));
+            Log::error(__METHOD__ . ' - ' . get_class($t) . ' - ' . $t->getMessage());
```

Comment:
Why not use the second param (context assoc array)? If this message is different every time then Sentry won't group it.